gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/ResourcesTest.kt

```diff
@@ -683,4 +680,22 @@ class ResourcesTest : GradlePluginTestBase() {
             }
         }
     }
+
+    @Test
+    fun checkTestResources() {
+        with(testProject("misc/testResources")) {
+            gradle("check").checks {
+                check.logContains("Configure main resources for 'desktop' target")
+                check.logContains("Configure test resources for 'desktop' target")
+                check.logContains("Configure main resources for 'iosX64' target")
+                check.logContains("Configure test resources for 'iosX64' target")
+                check.logContains("Configure main resources for 'iosArm64' target")
+                check.logContains("Configure test resources for 'iosArm64' target")
+                check.logContains("Configure main resources for 'iosSimulatorArm64' target")
+                check.logContains("Configure test resources for 'iosSimulatorArm64' target")
+
+                check.taskSuccessful(":desktopTest")
```

Comment:
I run common "check" task, so ios tests should be run on macos machines: https://github.com/JetBrains/compose-multiplatform/actions/runs/10564736433/job/29267782682?pr=5122#step:4:360

I don't run web tests because it requires Chrome.  
I don't run android tests because it requires new version of the resource's library.

Reply:
Web tests with resources don't work at all (a known test framework issue)